pyproject.toml

```diff
@@ -117,6 +117,7 @@ strict = true
 disallow_incomplete_defs = false
 disallow_untyped_defs = false
 disallow_untyped_calls = false
+exclude = ["examples/", "site/"]
```

Comment:
>Also cleaned up mypy config by excluding examples/

I don't get the point on why you excluded examples this time. can you elaborate more on that?

Reply:
Thanks, I wanted to avoid the exclusition. The current number of skip comments is fine